zigup.zig

```diff
@@ -200,8 +201,7 @@ pub fn main2() !u8 {
     if (args.len == 0) {
         help();
         return 1;
-    }
-    if (std.mem.eql(u8, "fetch-index", args[0])) {
+    } else if (std.mem.eql(u8, "fetch-index", args[0])) {
```

Comment:
The elses are unnecessary here, since each of these code paths return from the function.